dependencies/src/main/resources/linkage-checker-exclusion-default.xml

```diff
@@ -57,4 +57,18 @@
       https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/407
     </Reason>
   </LinkageError>
+  <LinkageError>
+    <Target>
+      <Package name="com.sun.jna" />
+    </Target>
+    <Source>
+      <Package name="reactor.blockhound.shaded.net.bytebuddy.agent.VirtualMachine" />
+    </Source>
+    <Reason>
+      BlockHound's shades ByteBuddy's VirtualMachine class. The class uses Java Native Access (JNA)
+      only when the class is available by catching ClassNotFoundException. Therefore the invalid
```

Comment:
If it's guarding the class, the rule above should reference the class (or classes) rather than a package.

Reply:
Updated this explanation to mention JNA package.